src/ray/protobuf/gcs_service.proto

```diff
@@ -336,6 +347,8 @@ service ObjectInfoGcsService {
       returns (GetAllObjectLocationsReply);
   // Add location of object to GCS Service.
   rpc AddObjectLocation(AddObjectLocationRequest) returns (AddObjectLocationReply);
+  // Add spilled URL location of object to GCS Service.
```

Comment:
Instead of adding a new RPC, why not extend the existing object location add request?

Reply:
No reason, I think either is okay. If you feel strongly about it, I can do that.

Reply:
Actually, I prefer it to be in the same RPC. Maybe not a blocker of the PR, but it looks cleaner to me (and easier to understand).